src/ingest_validation_tools/upload.py

```diff
@@ -1,38 +1,62 @@
+from __future__ import annotations
+
+import logging
+
+import os
+
+import subprocess
+from collections import Counter, defaultdict
 from datetime import datetime
-from collections import defaultdict
 from fnmatch import fnmatch
-from collections import Counter
 from pathlib import Path
-import subprocess
 
+import pandas as pd
+import requests
+from requests.auth import HTTPBasicAuth
+
+from ingest_validation_tools.plugin_validator import (
+    ValidatorError as PluginValidatorError,
+)
+from ingest_validation_tools.plugin_validator import run_plugin_validators_iter
+from ingest_validation_tools.schema_loader import PreflightError
 from ingest_validation_tools.validation_utils import (
-    get_tsv_errors,
-    get_data_dir_errors,
     dict_reader_wrapper,
     get_context_of_decode_error,
+    get_data_dir_errors,
+    get_directory_schema_versions,
     get_table_schema_version,
-    get_directory_schema_versions
-)
-
-from ingest_validation_tools.plugin_validator import (
-    run_plugin_validators_iter,
-    ValidatorError as PluginValidatorError
+    get_tsv_errors,
 )
 
-from ingest_validation_tools.schema_loader import (
-    PreflightError
-)
+TSV_SUFFIX = "metadata.tsv"
+API_KEY_SECRET = "d72052d0b8c37b286cb8ddfab63e8fad4069766fc6229af33e675f799b76a6d4"
```

Comment:
Make sure to remove this and make it configurable. Can't commit secret keys

Reply:
Ooooooops that's what I get for being lazy